### PR TITLE
Update gradle, kotlin, SDK

### PR DESCRIPTION
**Pull request by paolorotolo**
Changes:
 * Update gradle, kotlin version and SDK
 * Update deprecatd config in README

### CODE REVIEW

**Comment by paolorotolo** [comment]
Please test and merge.

File: build.gradle [hunk 1]
```diff
@@ -2,20 +2,21 @@
 buildscript {
     ext {
         gradleVersion = '3.1.1'
-        kotlinVersion = '1.2.31'
```
Comment (konaire) [inline build.gradle:5]: The latest version of kotlin is `1.3.11`. Can you put it instead?

File: build.gradle [hunk 2]
```diff
@@ -2,20 +2,21 @@
...
 
     repositories {
+        mavenCentral()
         google()
         jcenter()
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:$gradleVersion"
+        classpath "com.android.tools.build:gradle:3.2.1"
```
Comment (konaire) [inline build.gradle]: Can you put new version number in the constant and return the constant back?

**Review by konaire — CHANGES_REQUESTED** [review]
Great! Just 2 small issues to fix and I'll merge this PR 👍

File: build.gradle (hunk 1 above)
Comment (paolorotolo) [reply build.gradle:5]: Fixed in commit b9f9c60

**Comment by paolorotolo** [comment]
Done, please test again.

**Review by konaire — APPROVED** [review]
Well done 👍

**Pull request merged**